orga/app/components/stage-stars.js

```diff
@@ -21,4 +32,15 @@ export default class StageStars extends Component {
   get altMessage() {
     return this.intl.t('pages.assessment-individual-results.stages.value', { count: this.starsAcquired, total: this.starsTotal });
   }
+
+  get displayTooltip() {
+    return Boolean(this.withTooltip && this.reachedStage && (this.reachedStage.prescriberTitle || this.reachedStage.prescriberDescription));
+  }
+
+  get tooltipText() {
+    let text = this.reachedStage.prescriberTitle ? `<strong>${this.reachedStage.prescriberTitle}</strong>` : '';
+    text += this.reachedStage.prescriberTitle && this.reachedStage.prescriberDescription ? '<br/>' : '';
+    text += this.reachedStage.prescriberDescription ? `${this.reachedStage.prescriberDescription}` : '';
+    return htmlSafe(text);
+  }
```

Comment:
Ok ça répond à ma question précédente on peut avoir un palier sans titre et avec une description ;)

Reply:
Suggestion:
Plutôt que de conditionner le `<br>`, on pourrait utiliser un `<p>` sur la description.
```suggestion
  get tooltipText() {
    let text = this.reachedStage.prescriberTitle ? `<strong>${this.reachedStage.prescriberTitle}</strong>` : '';
    text += this.reachedStage.prescriberDescription ? `<p>${this.reachedStage.prescriberDescription}</p>` : '';
    return htmlSafe(text);
  }
```